staff/maider-hernandorena/arrays-native/scripts/join.test.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+describe('Array.prototype.join', function () {
+    var array ;
```

Comment:
You have a space between semicolon and the variable.

Reply:
Changed! :D